testsuite/MDAnalysisTests/__init__.py

```diff
@@ -98,11 +98,7 @@
 
 # keep in sync with RELEASE in setup.py
 __version__ = "0.19.1-dev"
-try:
-    from MDAnalysisTests.authors import __authors__
-except ImportError:
-    logger.info('Could not find authors.py, __authors__ will be empty.')
-    __authors__ = []
+from MDAnalysis import __authors__
```

Comment:
we never really had to dynamically generate this twice, since tests rely on main package